### PR TITLE
Adding font test.

### PR DESCRIPTION
**Pull request by tomgenoni**
This is a patch previous thought to have been merged but wasn't.

### CODE REVIEW

**Comment by tomgenoni** [comment]
@danoc please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 ## [Unreleased][unreleased]
 ### Added
 - [Patch] Add "Charcoal" color `#383838`.
+- [Patch] Add "Charcoal" color `#383838`.
```
Comment (danoc) [inline CHANGELOG.md]: This already exists.

**Comment by danoc** [comment]
Please fix the linter error in this PR or another one and then merge away!

**Comment by tomgenoni** [comment]
Fixing lint issue now.

**Pull request merged**